en/upgrades/nonembedded-codelist-changelog.rst

```diff
@@ -39,6 +39,11 @@ The :doc:`OrganisationRegistrationAgency </codelists/OrganisationRegistrationAge
      - Change description
      - Notes
      - Discussion
+   * - 25th February 2015
+     - :doc:`OrganisationRegistrationAgency </codelists/OrganisationRegistrationAgency>`
+     - Add *IN-MCA* (India - Ministry of Corporate Affairs)
```

Comment:
We added
code: IN-MCA
which has a name: Government of India, Minstry of Corporate Affairs

I don't see why a different name is being used here?